JitsiConference.js

```diff
@@ -1623,7 +1786,519 @@ JitsiConference.prototype.broadcastEndpointMessage = function(payload) {
 };
 
 JitsiConference.prototype.isConnectionInterrupted = function() {
-    return this.connectionIsInterrupted;
+    return this.p2pEstablished
+        ? this.isP2PConnectionInterrupted : this.isJvbConnectionIsInterrupted;
+};
+
+/**
+ * Handles {@link XMPPEvents.CONNECTION_INTERRUPTED}
+ * @param {JingleSessionPC} session
+ * @private
+ */
+JitsiConference.prototype._onIceConnectionInterrupted = function(session) {
+    if (session.isP2P) {
+        this.isP2PConnectionInterrupted = true;
+    } else {
+        this.isJvbConnectionIsInterrupted = true;
+    }
+    if (session.isP2P === this.p2pEstablished) {
```

Comment:
No it goes as: "if we're in P2P mode and event comes from P2P then emit the event" and vice versa.

There is no "jvbEstablished", because we don't care. There is confusion about P2P established, but I will rename that to "is P2P in use" like I've mentioned in the comment above.